projects/plugins/jetpack/class.jetpack.php

```diff
@@ -2035,15 +2051,20 @@ public function check_twitter_tags() {
 
 	/* Jetpack Options API */
 
+	/**
```

Comment:
Going to come back to this one until I fix everything under it so I don't mess up my spreadsheet of fixes. The CI does not show this as blocking.